### PR TITLE
Fix #222: Fix docs versioning and edit database structure docs

### LINKED ISSUE #222: Fix docs versioning and edit database structure docs

### PR DESCRIPTION
**Pull request by jandusil**
* Needs to be later backported

### CODE REVIEW

**Review by banterCZ — APPROVED** [review]
Looks good to me but let's wait also for review from @zcgandcomp

File: docs/Migration-Instructions.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 This page contains Mobile Utility Server migration instructions.
 
-- [Mobile Utility Server 1.5.0](./Mobile-Utility-Server-1.5.0.md)
+- [Mobile Utility Server 1.4.0](./Mobile-Utility-Server-1.4.0.md)
```
Comment (zcgandcomp) [inline docs/Migration-Instructions.md:5]: As we discuss originally. We cannot remove 1.5.0.
we need to have 1.4.0 guideline (as we backport the feature to 1.4.0)
AND 1.5.0 stating that there already can be migration.

**Review by zcgandcomp — CHANGES_REQUESTED** [review]

File: docs/Migration-Instructions.md (hunk 1 above)
Comment (jandusil) [reply docs/Migration-Instructions.md:5]: @zcgandcomp True, thanks. I have updated the 1.5.0 migration guide as per this comment https://github.com/wultra/mobile-utility-server/pull/174#pullrequestreview-1779184779

File: docs/Mobile-Utility-Server-1.5.0.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Migration from 1.4.x to 1.5.0
+
+This guide provides instructions for migrating from PowerAuth Mobile Utility Server version `1.4.x` to version `1.5.0`.
+
+No migration steps nor database changes are required.
```
Comment (zcgandcomp) [inline docs/Mobile-Utility-Server-1.5.0.md:5]: Please mention the backport that shift the original migration to 1.4.x migration. That's the issue. Liquibase, tags, our portal documentation original says migration from 1.1.x to 1.5.0. Ok we shift it so please put this information in the guideline.

**Review by zcgandcomp — CHANGES_REQUESTED** [review]

File: docs/Mobile-Utility-Server-1.5.0.md [hunk 3]
```diff
@@ -2,4 +2,8 @@
 
 This guide provides instructions for migrating from PowerAuth Mobile Utility Server version `1.4.x` to version `1.5.0`.
 
-No migration steps nor database changes are required.
\ No newline at end of file
+No migration steps nor database changes are required.
+
+Please note, version `1.4.x` was introduced as a backport of most functionalities from version `1.5.x`, due to specific
```
Comment (zcgandcomp) [inline docs/Mobile-Utility-Server-1.5.0.md:7]: Please note, due to a backport to the version `1.4.x`  database migration to 1.5.x was shifted and described in the guideline TBD link. 
If you are migrating from 1.4.0 there is no other migration needed. If you are migrating from 1.1.x. You need to perform migration steps to 1.4.x first.

**Review by zcgandcomp — CHANGES_REQUESTED** [review]
Rephrease.

Please backport the guideline so the portal will show the correct version for 1.5.x

File: docs/Mobile-Utility-Server-1.5.0.md (hunk 3 above)
Comment (jandusil) [reply docs/Mobile-Utility-Server-1.5.0.md]: I will prepare backport directly after merge

**Review by zcgandcomp — APPROVED** [review]
Looks OK.

**Pull request merged**